lib/pages/inquiry_page.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:dears/providers/inquiry_form_provider.dart';
+import 'package:dears/utils/theme.dart';
+import 'package:dears/widgets/inquiry_text_field.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class InquiryPage extends ConsumerWidget {
+  const InquiryPage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final enabled = ref.watch(
+      inquiryFormProvider.select((value) => value.enabled),
+    );
+
+    return Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        title: const Text("문의하기"),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
```

Comment:
literal number는 type inference에 의해 자동으로 `double`로 형변환되기 때문에 소수점을 붙이지 않아도 됩니다.

```suggestion
        padding: const EdgeInsets.all(16),
```